pkg/util/provider/machineutils/utils.go

```diff
@@ -47,6 +47,9 @@ const (
 	// MigratedMachineClass annotation helps in identifying machineClasses who have been migrated by migration controller
 	MigratedMachineClass = "machine.sapcloud.io/migrated"
 
+	// NotHanldedByMCM annotation helps in identifying the nodes which are not handled by MCM
+	NotManagedByMCM = "node.machine.sapcloud.io/notManagedByMCM"
```

Comment:
```suggestion
	NotManagedByMCM = "node.machine.sapcloud.io/not-managed-by-mcm"
```
_CamelCase_ may not be suitable for annotations.

Reply:
Noted, thanks ..will take care from next time.